collectd-plugins.yaml

```diff
@@ -94,6 +94,9 @@
   version: master
   repo: signalfx/vmstat-collectd
 
+- name: systemd
+  version: master
```

Comment:
Tag the latest commit you made to this repo and then use that version here.  Definitely don't use `master`.